src/main/java/org/codehaus/mojo/license/api/DefaultThirdPartyTool.java

```diff
@@ -624,13 +628,17 @@ public void overrideLicenses( LicenseMap licenseMap, SortedMap<String, MavenProj
             overrideMappings.load( overrideFile );
         }
 
+        VersionNames versionNames = new VersionNames("", "", "");
+        VersionRangeFactory versionRangeFactory = new VersionRangeFactory(versionNames);
+        NumericVersionFactory numericVersionFactory = new NumericVersionFactory(versionNames);
```

Comment:
Do we need duplicated version names in the file?

Reply:
I followed the example in the project's tests file, which suggests using a single `VersionNames` instance for both `VersionRangeFactory` and `NumericVersionFactory`. The example provided uses the same `VersionNames` object for both factories.

Could you please clarify if the concern is about the redundancy of passing the same instance, or if there's an expectation to separate the concerns by using different instances?

Reply:
We can refactor it as field to avoid code duplication